Sources/Apple/AppleAuthenticator+Models.swift

```diff
@@ -12,6 +12,7 @@ import Foundation
 public extension AppleAuthenticator {
   enum Nonce {
     case random(length: UInt)
+    case custom(value: String)
   }
```

Comment:
Maybe we can replace this enumeration with a function?
I.e. `typealias Nonce = () -> String`

Reply:
Good idea, but I think it's not necessary here. In almost all cases we are using `random` which is more convenient.